services/class-wc-ebanx-helper.php

```diff
@@ -5,14 +5,29 @@
 }
 
 require_once WC_EBANX_GATEWAYS_DIR . 'class-wc-ebanx-global-gateway.php';
+require_once WC_EBANX_VENDOR_DIR . 'autoload.php';
 
 use RuntimeException as RuntimeException;
+use Ebanx\Benjamin\Services\Gateways\CreditCard;
+use Ebanx\Benjamin\Models\Country;
 
 /**
  * Class WC_EBANX_Helper
  */
 abstract class WC_EBANX_Helper {
 
+	const COUNTRY_ABBREVIATION_BRAZIL = 'br';
```

Comment:
We should move this to the WC_EBANX_Constants class and also replace a similar code on the WC_EBANX_Global_Gateway.